includes/class-mastodon-api.php

```diff
@@ -2661,6 +2661,10 @@ public function api_account_statuses( $request ) {
 	public function api_account( $request ) {
 		$user_id = $this->get_user_id_from_request( $request );
 
+		if ( is_multisite() && ! is_user_member_of_blog( $user_id ) ) {
```

Comment:
`is_user_member_of_blog` [does a `is_multisite()` check](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/user.php#L1100-L1102).
```suggestion
		if ( ! is_user_member_of_blog( $user_id ) ) {
```

Reply:
Nice catch!

Reply:
Oh I didn't pay enough attention, I had actually broken this out as a dedicated PR https://github.com/akirk/enable-mastodon-apps/pull/158 since it wasn't strictly related to this already sizeable PR